lib/core/re.nit

```diff
@@ -366,19 +379,19 @@ class Regex
 
 		# Found one?
 		if res == 0 then
-			var bso = bstr.byte_to_char_index(native_match.rm_so)
-			var ln = bstr.byte_to_char_index(native_match.rm_eo - native_match.rm_so - 1)
+			var first_char = bstr.byte_to_char_index(native_match.rm_so)
+			var last_char = bstr.byte_to_char_index(native_match.rm_eo - native_match.rm_so - 1)
```

Comment:
I can change it to length... but something bugs me in the rest of the line then. It looks to me like `bstr.byte_to_char_index` uses `bstr` to count the number of chars until the given byte, not the numbers of char between 2 byte offset. So here, for a match of 4 bytes in length, it would return the number of chars since the beginning of the string up to 4 bytes, instead of using the `first_char` as starting point. @R4PaSs, what do you think?

I may be wrong, it's a bit hard to tell since `byte_to_char_index` is a private method without any doc... ... ...